variables.tf

```diff
@@ -480,6 +480,17 @@ variable "versioning_enabled" {
   description = "When set to 'true' the s3 origin bucket will have versioning enabled"
 }
 
+variable "bucket_versioning" {
```

Comment:
This effectively deprecates `versioning_enabled`. For backwards compatibility it should be mapped to the new variable with value "Enabled". 

It would also be good to remove its use from the examples.

Reply:
I might be misunderstanding, but since `var.versioning_enabled` is a `bool`, wouldn't I break backwards compatibility if I changed it to a string and set it's default to `Enabled`? Or do you mean to map it via something like doing some logic in `locals`?

Something like

```hcl
locals {
  bucket_versioning = var.versioning_enabled || var.bucket_versioning == "Enabled" ? "Enabled" : "Disabled"
}
```

and then use `local.bucket_versioning` everywhere instead?